connexion/decorators/response.py

```diff
@@ -44,10 +43,10 @@ def validate_response(self, data, status_code, headers):
             try:
                 # For cases of custom encoders, we need to encode and decode to
                 # transform to the actual types that are going to be returned.
-                data = json.dumps(data)
-                data = json.loads(data)
+                data = data.replace(b'\\"', b'"')
```

Comment:
We need to handle encoding properly. Could you please elaborate on what is the real problem here?